nebula-unity/Assets/scripts/DungeonMapGenerator.cs

```diff
@@ -5,7 +5,9 @@
 // All generated rooms will be an odd number of tiles in at least one dimension to allow for a central door
 public class DungeonMapGenerator : MonoBehaviour {
     [Header("Physical prefabs for generation objects")]
-    public GameObject floorTile;    // Prefab of a floor tile model to generate.  
+    public GameObject floorTile;   // Prefab of a floor tile model to generate.  
+    public GameObject wallTile;    // Prefab of a wall tile model to generate.  
+    public GameObject doorTile;    // Prefab of a door tile model to generate.  
```

Comment:
This is actually temporary.  This will allow for fast iterations of swapping out tiles to check artwork since it is just a drag+drop in the inspector.  Looking forward, I think it makes the most sense to leave these as testing options, but as we get more art, dynamically looking into specific folders for the assets instead of forcing you to drag them all into the inspector.